schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE ANIMALS(
    DATE_OF_BIRTH           DATE     NOT NULL,
    ESCAPE_ATTEMPTS         INT      NOT NULL,
    NEUTERED                BOOLEAN  NOT NULL,
-   WEIGHT_KG               DECIMAL  NOT NULL
+   WEIGHT_KG               DECIMAL  NOT NULL,
+   SPECIES                 VARCHAR(100)
```

Comment:
Thanks for the comment, I did use ALTER I just thought that the purpose is to update code the in the schema instead of putting what I did, my bad I will update it accordingly.